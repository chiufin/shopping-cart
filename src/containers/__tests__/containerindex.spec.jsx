import React from 'react'
import { shallow } from 'enzyme'
import { HomeContainer } from '..'

describe('Container Mocking', () => {
  it('HomeContainer Mocks correctly', () => {
    expect(HomeContainer).toMatchSnapshot()
    const shallowHomeContainer = shallow(<HomeContainer />)
    expect(shallowHomeContainer).toHaveLength(1)
  })
})
