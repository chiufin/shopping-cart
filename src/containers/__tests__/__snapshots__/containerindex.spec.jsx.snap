// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Mocking HomeContainer Mocks correctly 1`] = `[Function]`;
